hexrd/ui/main_window.py

```diff
@@ -747,6 +747,14 @@ def update_config_gui(self):
 
     def eventFilter(self, target, event):
         if type(target) == QMainWindow and event.type() == QEvent.Close:
+            msg = 'Are you sure you want to quit?'
+            buttons = (QMessageBox.No | QMessageBox.Yes)
+            default = QMessageBox.No
+            response = QMessageBox.question(
+                self.ui, 'HEXRD', msg, buttons, default)
```

Comment:
It looks to me that the `buttons` and `default` arguments are the same as the defaults for `QMessageBox.question()`. Maybe we can remove them?

Reply:
Yes, thank you. Removed.